dist/about.html

```diff
@@ -28,69 +28,72 @@
+      <p class="about-hero__contact-us">Please contact me per Email for any further questions about Animation Fans 2022 <span class="about-hero__contact-us__email">info@animationfans.com</span></p>
       
     </section>
     <hr class="anifans-logo__divider">
     <section class="anifans-logo lato_regular">
-      <h2 class="anifans-logo__title">Creative Commons Global Summit 2015 Logo</h2>
+      <h2 class="anifans-logo__title">Animation Fans Logo Explained</h2>
       <hr class="about-divider">
       <p class="anifans-logo__text-content">
-        Lorem ipsum dolor sit amet consectetur adipisicing elit. Hic vero molestiae, vitae veritatis ipsa obcaecati est corporis architecto temporibus nisi.
+        For the logo I just went for two single concepts, minimalism and cuteness. Everyone, or almost everyone, love cute things, so I decided to go for something that was recognizable and cute, thus, I picked a "moe" anime girl to be the face of the site for the time being, also, a modern website/brand needs a modern logo, so I went for a minimalist, simple and easy design, using Montserrat font for it.
       </p>
       <img src="./img/logo/logo-black.png" alt="" class="anifans-logo__logo">
     </section>
     <hr class="anifans-logo__divider">
     <section class="past-expo lato_regular">
-      <h2 class="past-expo__title">See the past CC Global Summits</h2>
+      <h2 class="past-expo__title">Last Expo I visited!</h2>
       <hr class="about-divider">
-      <p class="past-expo__text-content">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Harum quibusdam eos, eaque ipsa quidem at autem suscipit aliquid illo voluptatibus!</p>
+      <p class="past-expo__text-content">Saudi Anime Expo was the last expo I had the chance to assist. Since I could not assist Los Angeles Anime Expo, I went all the way to Saudi Arabia, it was so much fun!</p>
       <div class="past-expo__grid">
         <div class="past-expo__grid-item__item1">
-          <h3 class="past-expo__grid-item__item1__title">2013</h3>
-          <p class="past-expo__grid-item__item1__text-content">CC Global Summit 2013 in Buenos Aires</p>
+          <h3 class="past-expo__grid-item__item1__title">2022</h3>
+          <p class="past-expo__grid-item__item1__text-content">CC Mangakas at Saudi Anime Expo telling us about their upcoming anime projects</p>
         </div>
         <div class="past-expo__grid-item__item2">
-          <h3 class="past-expo__grid-item__item1__title">2013</h3>
-          <p class="past-expo__grid-item__item1__text-content">CC Global Summit 2011 in Warsaw</p>
+          <h3 class="past-expo__grid-item__item1__title">2022</h3>
+          <p class="past-expo__grid-item__item1__text-content">One Piece conference, I love One Piece, but I think I will be old when Luffy finds it!</p>
         </div>
       </div>
     </section>
   </main>
   <footer class="footer lato_regular">
```

Comment:
- [ ] In mobile view, the footer contents are not displayed according to the given template. Kindly address this.
- [ ] Also the background color is white with black text, address this as well.
- [ ] The correct font has not been applied to this section. Please apply it.

Your design
![image](https://user-images.githubusercontent.com/81932478/169600183-3d62213d-0736-41bd-90e2-b27ad30ae795.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/169600254-21c74bdd-3687-4798-89f0-d1ac4210792c.png)